src/Language/ASKEE/Model.hs

```diff
@@ -213,3 +214,13 @@ printModel m =
     printJson v = BS.unpack $ JSON.encode v
 
 
+modelUID :: Model -> Either String Text
+modelUID m =
+  case m of
+    Easel esl -> Right $ ESL.modelName esl
```

Comment:
feel free to ignore this as it is incredibly petty but i don't think Easel name should be considered as a UID - i think UIDs are gromet specific and perhaps this function (with its current type and implemented for gromets only) belongs in the Gromet namespace somewhere implemented for gromets only where the other types say something you can't do <whatever> because you're trying to do it to something that's not a gromet

Reply:
Yeah, I agree with your point on Easel names. I can't put this function into the most general gromet namespace, `L.A.Gromet`, since that would produce a cyclic dependency between it and `L.A.Model`. Regardless, I'll make it so that non-gromets produce a `Left` value.